Section 2/2.48_Mixins_3_Modules.rb

```diff
@@ -0,0 +1,38 @@
+require 'bundler/inline'
+ 
+gemfile true do
+ source 'http://rubygems.org'
+ gem 'bcrypt'
+end
+ 
+module Crud
+  require 'bcrypt'
+  puts "Module CRUD activated"
+ 
+  # When you call a method with its class name like this, 
+  # it's called a class method, and you can also use self.creaate_secure_users here, 
+  # instead of Crud.create_secure_users
+  def Crud.create_hash_digest(password)
+    BCrypt::Password.create(password)
+  end
+ 
+  def Crud.verify_hash_digest(password)
+    BCrypt::Password.new(password)
+  end
```

Comment:
This is not verifying. This is just getting the hashed value of the string passed in.
Either rename the method or refactor the method to reflect that it means what the name says it does